src/index.js

```diff
@@ -842,7 +858,15 @@ async function start() {
       case "users": {
         // "!hubs users" == list users
         if (hubState != null) {
-          const names = Object.values(hubState.reticulumCh.getUsers()).map(info => info.metas[0].profile.displayName);
+          const names = Object.values(hubState.reticulumCh.getUsers()).map(
+            info => {
+              let profile = info.metas[0].profile;
```

Comment:
Likewise, Optional Chaining looks like it would be useful here.

Reply:
It does look like there is the possibility that `metas` could be empty at some point, so I guess it should help.  I'm slightly worried about this masking errors and giving an incomplete report (which wouldn't be great for moderation), but potentially crashing the bot because a user is in a transitionary state is probably worse than an incomplete report.  Updated